### ISSUE TITLE
rpm package for opensuse leap 15.1?

### ISSUE THREAD

**Issue by nevgeniev** [thopiekar/rcraid-dkms#20]
hi,

trying to install linux on dell aurora r10 ryzen edition  (3700x with x570 chipset) opensuse leap 15.1 turns out to be the only installation media able to boot on this machine w/o issues. well the only problem is ... rcraid :) 

are there instructions on how to build, sideload rcraid-dkms for open suse, or prebuilt packages?

**Comment by thopiekar** [comment]
Nope, you will need to follow the instructions by AMD. However, I think they don't use DKMS afaik.
You can also make a dkms-enabled RPM package, too. However, as I'm on Debian/Ubuntu and don't use any RPM-based OS, I don't have interest to spend time on this.

Feel free to contribute!

**Comment by tian2992** [comment]
I'm also interested on OpenSUSE Support. Any suggested instructions generic to any OS that can serve as a starting point?
